docs/src/HowToGuides/BalanceLaws/how_to_define_bcs.md

```diff
@@ -0,0 +1,52 @@
+# How to define boundary conditions
+
+```@meta
+CurrentModule = ClimateMachine.BalanceLaws
+```
+
+## Discontinuous Galerkin (DG) method
+
+Boundary conditions (BCs) are defined _weakly_ in ClimateMachine's
+DG implementation. We do this by defining BCs _through_ the _fluxes_,
+rather than on the prognostic variables themselves. Therefore, BCs
+are enforced by prescribing fluxes (numerical fluxes, in DG terminology)
+that drive the solution at the boundary to satisfy our desired BCs.
+
+One major advantage of this implementation approach is that BCs are
+handled in phase space, so we only need one implementation for all
+time steppers.
+
+In the context of the PDE that `ClimateMachine.jl` solves:
+
+`R(Y, t) = ∂_t Yᵢ + (∇•(F₁(Y) + F₂(Y,G)))ᵢ - (S(Y,G))ᵢ = 0, G = ∇Y`,
+
+the following functions must be defined for a
```

Comment:
"_must_ be defined" feels at odds with the "If present" column in the table. Perhaps something like "The terms in a balance law determine what functions must be defined for the BCs..." and making the "If present" column the second column?

Reply:
Agreed, this needs more work.